task1/task1.py

```diff
@@ -0,0 +1,45 @@
+"""Task1: some examples."""
+import re
+
+# line 1
+
+print('Hello, world!\n')
+
+# line 2
+
+name = input('What is your name?')
+print('Hi, %s.\n' % name)
+
+# line 3
+
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print("iteration {iteration} is {name}".format(iteration=i, name=name))
+
+# line 4
+
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+# line 5
+
+
+def greet(name):
+    """Print my name."""
+    print('Hello', name)
+
+
+greet('Jack')
+greet('Jill')
+greet('Bob')
+
+# line 6
+
```

Comment:
It's OK, but I'm confused with blank lines. Here you have one blank line before block comment and two after and not vise versa. In other places we have 1\1 blank lines. More of that we have blank lines after all comments but don't have it after docstring. 
You shouldn't fix it, but as for me it would be more beautiful to have same comments within a file.